ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -14,19 +23,36 @@ function join_by { local IFS="$1"; shift; echo "$*"; }
 
 REGION="${LEASED_RESOURCE}"
 # BootstrapInstanceType gets its value from pkg/types/aws/defaults/platform.go
+architecture="amd64"
 BOOTSTRAP_NODE_TYPE=m5.large
+if [[ "${CLUSTER_TYPE}" == "aws-arm64" ]]; then
+  architecture="arm64"
+  BOOTSTRAP_NODE_TYPE=m6g.large
+fi
 
 workers=3
 if [[ "${SIZE_VARIANT}" == "compact" ]]; then
   workers=0
 fi
 master_type=null
 if [[ "${SIZE_VARIANT}" == "xlarge" ]]; then
-  master_type=m5.8xlarge
+  if [[ "${CLUSTER_TYPE}" == "aws-arm64" ]]; then
+    master_type=m6g.8xlarge
+  else
+    master_type=m5.8xlarge
```

Comment:
maybe cleaner to define earlier e.g. `arch_instance_type=m5` or `m6g` depending on `CLUSTER_TYPE`, then just change these definitions to `master_type=${arch_instance_type}.8xlarge` etc.?